chart/values.yaml

```diff
@@ -49,14 +28,19 @@ controllerManager:
 kubernetesClusterDomain: cluster.local
 metricsService:
   ports:
-  - name: https
+  - name: metrics
     port: 8443
     protocol: TCP
-    targetPort: https
+    targetPort: metrics
   type: ClusterIP
 webhookService:
   ports:
   - port: 443
     protocol: TCP
     targetPort: 9443
   type: ClusterIP
+
+leaderWorkerSet:
```

Comment:
We can't change the chart directly unless the values.global.yaml, because they're auto-generated by `make helm`.